app/car/car_class.py

```diff
@@ -0,0 +1,23 @@
+
+class Car:
+    def __init__(self, car: dict, fuel_price: float) -> float:
+        self.brand = car["brand"]
+        self.fuel_consumption = car["fuel_consumption"]
+        self.fuel_price = fuel_price
+
+    def cost_of_way(
+            self,
+            customer_location: list,
+            shop_location: [list]
+    ) -> float:
+        distance = pow(
+            pow(shop_location[0] - customer_location[0], 2)
+            + pow(shop_location[1] - customer_location[1], 2),
+            (1 / 2)
+        )
+        cost_of_distance = round(
+            (distance * 2 * self.fuel_price * self.fuel_consumption) / 100,
+            2
+        )
+
+        return round(cost_of_distance, 2)
```

Comment:
There is no need for these two consecutive round operations, you should allocate a variable to write all calculation results there and then round it once in the `return` statement.